src/platforms/platform.dom.css

```diff
@@ -0,0 +1,51 @@
+/*
+ * DOM element rendering detection
+ * https://davidwalsh.name/detect-node-insertion
+ */
+@-webkit-keyframes chartjs-render-animation {
```

Comment:
It looks like Chrome 42 was the last version to require this. I think it'd be pretty safe to remove now, which would give us smaller download sizes